app/models/user.rb

```diff
@@ -527,6 +528,10 @@ def conditionally_validate_summary
     errors.add(:summary, "is too long.") if summary.present? && summary.size > 200
   end
 
+  def evaluate_markdown
+    self.summary_html = MarkdownParser.new(summary).evaluate_limited_markdown
```

Comment:
This should  run only if summary is present

Reply:
See the previous comment, the callback needs to be executed, is the parser that should be called only if the summary is populated, otherwise summary_html needs to be set to nil